pages/common/radare2.md

```diff
@@ -0,0 +1,28 @@
+# radare2
+
+> A set of reverse engineering tools.
+> More information: <https://radare.gitbooks.io/radare2book/>.
+
+- Open a file in write mode without parsing the file format headers.:
+
+`radare2 -nw {{binary}}`
+
+- Debug a program:
+
+`radare2 -d {{binary}}`
+
+- Run a script before entering interactive command-line prompt:
+
+`radare2 -i {{script.r2}} {{binary}}`
+
+- Show a help page of any radare2 command within the interactive client:
+
+`> {{radare2_command}}?`
+
+- Run a shell command from interactive radare:
```

Comment:
Perhaps this will read better?
```suggestion
- Run a shell command from the interactive CLI:
```

Why is the `!` required here?

Reply:
Thank you for your review.

Since this command is within the interactive CLI, the user is not in a regular shell. So to be able to execute **shell** commands from the interactive CLI, the `!` is needed here.
For example if the user wants to look up a file within the CLI, he does not have to exit the CLI, but can just do the following:
`> !ls`
which will execute the shell command `ls`.

Reply:
Ah, I see now. Thanks for the clarification :-)